aspnetcore/grpc/performance.md

```diff
@@ -156,6 +156,32 @@ The preceding code configures a channel that sends a keep alive ping to the serv
 
 :::moniker-end
 
+## Flow control
+
+HTTP/2 flow control is a feature that prevents apps from being overwhelmed with data. When using flow control:
+
+* Each app has an available buffer window. The buffer window is how much data the app can receive at once.
+* Flow control activates if the buffer window is filled up. When activated, the sending app pauses sending more data.
+* Once the receiving app has processed data, then space in the buffer window is available. The sending app resumes sending data.
+
+Flow control can have a negative impact on performance when sending and receiving large messages. If the buffer window is smaller than incoming message payloads, or there's latency between the client and server, then data can be sent in start/stop bursts.
```

Comment:
The comma before the "or" is breaking the two objects of the conditional clause ("if"). 

```suggestion
Flow control can have a negative impact on performance when sending and receiving large messages. If the buffer window is smaller than incoming message payloads or there's latency between the client and server, then data can be sent in start/stop bursts.
```